### PR TITLE
Support Coverity Scan from GitHub Actions

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
Inspired by:
https://github.com/ruby/actions-coverity-scan

### CODE REVIEW

**Comment by DimitriPapadopoulos** [comment]
It works on my own [`coverity_scan`](https://github.com/DimitriPapadopoulos/openfortivpn/tree/coverity_scan) branch.

We can test against the main repository once the secret token `COVERITY_SCAN_TOKEN` has been set.

File: .github/workflows/coverity-scan.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+name: Synopsys
+
+on:
+  push:
+    branches: [coverity_scan]
```
Comment (adrienverge) [inline .github/workflows/coverity-scan.yml:6]: Is it a leftover?

**Review by adrienverge — COMMENTED** [review]
`COVERITY_SCAN_TOKEN` has been set! :slightly_smiling_face:

File: .github/workflows/coverity-scan.yml (hunk 1 above)
Comment (DimitriPapadopoulos) [reply .github/workflows/coverity-scan.yml:6]: We are limited to a few scans per week/day:
[What is the frequency for build submissions to Coverity Scan?](https://scan.coverity.com/faq#frequency)

We have been checking only the `coverity_scan` branch, which we update manually.

Comment (adrienverge) [reply .github/workflows/coverity-scan.yml:6]: Ah OK, I missed that. Perfect then!

**Pull request closed without merging**